### PR TITLE
chore: seperate progress components

### PR DESCRIPTION
**Pull request by dammy001**

### CODE REVIEW

File: packages/components/progress/src/progressIndicator.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  MergeProps,
+  PrimitiveProps,
+  RefElement,
+} from '@oku-ui/primitive'
+import type { Scope } from '@oku-ui/provide'
+import { useRef } from '@oku-ui/use-composable'
+import { getProgressState } from './utils'
+import { useProgressContext } from './progress'
+
+const INDICATOR_NAME = 'ProgressIndicator'
```
Comment (productdevbook) [inline packages/components/progress/src/progressIndicator.ts:14]:
```suggestion
const INDICATOR_NAME = 'OkuProgressIndicator'
```

File: packages/components/progress/src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const PROGRESS_NAME = 'Progress'
```
Comment (productdevbook) [inline packages/components/progress/src/constants.ts:1]: why here component name

Comment (dammy001) [reply packages/components/progress/src/constants.ts:1]: The constant is reused in some methods in utils so let's just create a file that hold  the constants

Comment (productdevbook) [reply packages/components/progress/src/constants.ts:1]: hmm okay `const INDICATOR_NAME = 'OkuProgressIndicator' but why dont add ?

File: packages/components/progress/src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (productdevbook) [inline packages/components/progress/src/constants.ts:1]:
```suggestion
export const PROGRESS_NAME = 'OkuProgress'
```

**Review by productdevbook — APPROVED** [review]

**Pull request merged**